### DISCUSSION TITLE
Problem with my school email and 2FA keys.

### DISCUSSION THREAD

**Discussion by AlexM4Rewrite** [community/community#136982] [GitHub Education]
### Hi everyone,

So, I made an account called "Alex4Hire," and I had a 2FA app(Ravio). 
Ravio had an update that deleted all my previous tokens(along with Github) and I have to switched over to another app.
I didn't have any prior recovery codes at time with my 2FA.
And I had to unlinked my school email and make another account(which is this one).
I'm not eligible with the Github Education, because of the same email domain. 
What should I do?

**Comment by davevad93** [comment]
Hi @Al3jandoDiaz7-4-3 ,

If your former account has the `Student Developer Pack`, unfortunately you can't reapply with the same educational email using your current account. Student benefits are non-transferable, you can merge two accounts, but the account with the benefits would be the main account.

You can try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue.
